codegen/src/main/java/software/amazon/awssdk/codegen/model/config/customization/CustomizationConfig.java

```diff
@@ -123,6 +123,8 @@
 
     private Map<String, String> modelMarshallerDefaultValueSupplier;
 
+    private boolean useAutoConstructList = false;
```

Comment:
This should default to true.

Reply:
Any reason why? It's default false in v1: https://github.com/aws/aws-sdk-java/blob/7bf266202554a33d1f1e89d1691a5a389d36966c/aws-java-sdk-code-generator/src/main/java/com/amazonaws/codegen/model/config/customization/CustomizationConfig.java#L41

Reply:
Cool, wasn't aware of that but cool with it! Will update

Reply:
Updated to make auto construct lists the default